<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Sinc-интерполятор</string>
  </property>
  <property name="windowIcon">
   <iconset resource="q.qrc">
    <normaloff>:/icon/icon.ico</normaloff>:/icon/icon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="inFileLBL">
     <property name="text">
      <string>Выберите входной файл:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="inFileLED">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="inFilePBN">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="q.qrc">
         <normaloff>:/folder/folder.png</normaloff>:/folder/folder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="outFileLBL">
     <property name="text">
      <string>Выберите выходной файл:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLineEdit" name="outFileLED">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Если не выбран, файл будет создан в папке исходного</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="outFilePBN">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="q.qrc">
         <normaloff>:/folder/folder.png</normaloff>:/folder/folder.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="outFileCancel">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="q.qrc">
         <normaloff>:/cross/cross.png</normaloff>:/cross/cross.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="optionsGRB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Настройки интерполятора</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="samplingInLBL">
          <property name="text">
           <string>Частота дискретизации исходного файла</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="samplingInSPB">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>512000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="samplingInHSL">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>512000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="samplingOutLBL">
          <property name="text">
           <string>Частота дискретизации выходного файла</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="samplingOutSPB">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>512000</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="samplingOutHSL">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>512000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="extraSettingsCHB">
        <property name="text">
         <string>Дополнительные настройки (Не применяются, если выключены)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="blockSizeLBL">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Размер блока (в семплах)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="blockSizeSPB">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>2048</number>
          </property>
          <property name="value">
           <number>64</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSlider" name="blockSizeHSL">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>2048</number>
          </property>
          <property name="value">
           <number>64</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="overlapLBL">
          <property name="text">
           <string>Размер перекрытия (в семплах)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="overlapSPB">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>1023</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="overlapHSL">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>1023</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="mainPBN">
     <property name="text">
      <string>Обработать</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="infoLBL">
     <property name="text">
      <string>Обрабатываю файл</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="infoPRB">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="q.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>blockSizeHSL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blockSizeSPB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blockSizeSPB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blockSizeHSL</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>samplingInHSL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>samplingInSPB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>samplingOutHSL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>samplingOutSPB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>samplingInSPB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>samplingInHSL</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>samplingOutSPB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>samplingOutHSL</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>overlapSPB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>overlapHSL</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>overlapHSL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>overlapSPB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
